README.md

```diff
@@ -55,13 +61,29 @@ services:
         class: Tarifhaus\Doctrine\ORM\NullableEmbeddableListener
         arguments:
             - '@property_accessor'
+            - '@property_accessor'
         calls:
             - ['addMapping', ['Thengine\Domain\User\Model\UserProfile', 'address']]
         tags:
             - { name: 'doctrine.orm.entity_listener', entity: '\Thengine\Domain\User\Model\UserProfile', event: 'postLoad' }
 ```
 
-# Usage
+There is also a `nullator` using a bound closure to set the `null` value privately.
 
-The `embeddable` class needs to implement the `Tarifhaus\Doctrine\ORM\NullableEmbeddableInterface`.
-It contains a method `isNull(): bool` which tells the listener, whether the loaded embeddable should be treated as and therefore replaced with `null`.
+```yaml
+services:
+    tarifhaus.doctrine.nullable_embeddable_closure_nullator:
+        public: false
+        class: Tarifhaus\Doctrine\ORM\NullableEmbeddable\ClosureNullator
+
+    tarifhaus.doctrine.nullable_embeddable_listener:
+        public: false
+        class: Tarifhaus\Doctrine\ORM\NullableEmbeddableListener
+        arguments:
+            - '@property_accessor'
+            - '@tarifhaus.doctrine.nullable_embeddable_closure_nullator'
+        calls:
+            - ['addMapping', ['Thengine\Domain\User\Model\UserProfile', 'address']]
+        tags:
+            - { name: 'doctrine.orm.entity_listener', entity: '\Thengine\Domain\User\Model\UserProfile', event: 'postLoad' }
```

Comment:
Adding configuration for Symfony4 would be nice :3 (implies many lines)

Reply:
What exactly do you mean? What's missing?